keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/KeanuLapack.java

```diff
@@ -0,0 +1,67 @@
+package io.improbable.keanu.tensor.dbl;
+
+import lombok.AllArgsConstructor;
+
+import static org.bytedeco.javacpp.openblas.LAPACKE_dgetrf;
+import static org.bytedeco.javacpp.openblas.LAPACKE_dgetri;
+import static org.bytedeco.javacpp.openblas.LAPACKE_dpotrf;
+import static org.bytedeco.javacpp.openblas.LAPACK_ROW_MAJOR;
+
+/**
+ * https://software.intel.com/en-us/mkl-developer-reference-c-matrix-factorization-lapack-computational-routines
+ * <p>
+ * This is a wrapper around a few LAPACK functions used directly in Keanu
+ */
+public class KeanuLapack {
```

Comment:
can you give the method arguments more descriptive names? Not just the arbitrary ones like `N`, `n`, but even `ipiv` would be better as `pivotIndices`